makelib/golang.mk

```diff
@@ -18,6 +18,11 @@
 # The go project including repo name, for example, github.com/rook/rook
 GO_PROJECT ?= $(PROJECT_REPO)
 
+# the packages to be built statically, for example, $(GO_PROJECT)/cmd/mytool
+ifeq ($(GO_STATIC_PACKAGES),)
+$(error please set GO_STATIC_PACKAGES prior to including golang.mk)
+endif
+
```

Comment:
why was this re-added? I think it was taken out to support projects that don't build any packages.